client/app/components/packages/deis/edit.hbs

```diff
@@ -27,6 +27,16 @@
           {{optionset 'package' 'statuscode' 'label' @package.statuscode}}
         </p>
       </section>
+      
+      {{#if @package.dcpPackagenotes}}
+        <section class="form-section">
```

Comment:
Please use saveableForm.Section instead. See the Draft EAS edit component

Reply:
https://github.com/NYCPlanning/labs-applicant-portal/pull/815/files

Reply:
Thanks @godfreyyeung , done.